File: Python/Calculadora_de_Bhaskara.py
#Autor: Pedro Gonella Barão
#Data: 24/032025
#Objetivo: Calculadora de Bhaskara

import math


print("\nBem-vindo a Calculadora de Bhaskara!")
a = float(input("Qual é o valor de a?: "))
b = float(input("Qual é o valor de b?: "))
c = float(input("Qual é o valor de c?: "))

x1 = (-b + math.sqrt((b**2) - 4 * a * c))/(2 * a)
x2 = (-b - math.sqrt((b**2) - 4 * a * c))/(2 * a)

print(f"\nO valor de x1 é igual a {x1}")
print(f"O valor de x2 é igual a {x2}")

File: C/Calculadora_de_Bhaskara.c
/*

    Autor: Pedro Gonella Barão
    Data: 21/032025
    Objetivo: Calculadora de Bhaskara

*/

#include <stdio.h>
#include <math.h>


int main()
{
    float a;
    float b;
    float c;
    float x1;
    float x2;

    printf("\nBem-vindo(a) calculadora de Bhaskara!\n");
    printf("Qual seria o valor de a?: ");
    scanf("%f", &a);
    printf("Qual seria o valor de b?: ");
    scanf("%f", &b);
    printf("Qual seria o valor de c?: ");
    scanf("%f", &c);


    x1 = (-b + sqrt(pow(b, 2) - 4 * a * c))/(2 * a);
    x2 = (-b - sqrt(pow(b, 2) - 4 * a * c))/(2 * a);

    printf("\nO valor de x1 e de = %.3f", x1);
    printf("\nO valor de x2 e de = %.3f", x2);

    return 0;
}